tests/lib/__init__.py

```diff
@@ -100,6 +100,10 @@ def packages2(self):
     def packages3(self):
         return self.root.join("packages3")
 
+    @property
+    def packages4(self):
```

Comment:
This feels unnecessary -- why has a simple package been added under a new, different directory; am I missing something?

Reply:
Some tests expect the source distribution of `simple` to be used. Once PEP 518 support using source distributions for the build requirements `package4` can be removed.

Reply:
Ah, Okay. :)